src/main/kotlin/test/ktortemplate/core/httphandler/DefaultRoutes.kt

```diff
@@ -27,11 +28,13 @@ fun Route.defaultRoutes() {
         val totalElements = carService.count(pageRequest)
         val data = carService.list(pageRequest)
         call.respond(
-            PageResponse.from(
-                pageRequest = pageRequest,
-                totalElements = totalElements,
-                data = data,
-                path = call.request.path()
+            JsonSettings.toJson(
```

Comment:
It this call to `.toJson()` required? `PageResponse` is already `@Serializable`, why is this needed here?